src/components/aktivitetskrav/vurdering/ForhandsvarselOppsummering.tsx

```diff
@@ -29,9 +30,10 @@ export const ForhandsvarselOppsummering = ({
         <Tag variant="warning-moderate">{`Frist: ${fristDato}`}</Tag>
       </div>
       {beskrivelse && (
-        <Panel className="border-gray-400 rounded p-4 max-h-24 overflow-y-auto">
-          <BodyShort>{beskrivelse}</BodyShort>
-        </Panel>
+        <ExpandableBlockquote
+          className="border-gray-400 rounded p-4"
+          body={beskrivelse}
```

Comment:
Hadde kanskje vært mer om beskrivelse ble sendt inn som children her i stedet for `body`-prop?